easybuild/easyconfigs/h/HTSlib/HTSlib-1.9-intel-2018b.eb

```diff
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+# 1.4 modified by:
+# Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'ConfigureMake'
```

Comment:
@boegel isn't this already included in a slightly different form in #7322 ?

Reply:
Yes it is, let's get that one merged first...